mercury.spec

```diff
@@ -77,6 +78,11 @@ cmake -DMERCURY_USE_CHECKSUMS=OFF                \
       -DMERCURY_ENABLE_VERBOSE_ERROR=ON          \
       -DBUILD_TESTING=OFF                        \
       -DNA_USE_OFI=ON                            \
+      -DNA_USE_UCX=ON                            \
```

Comment:
Just curious but I got the impression from previous discussions that OFI and UCX could not be enabled at the same time in Mercury, that it would be built for one or the other. Maybe I was mistaken?

Reply:
It is not a problem to enable both concurrently.  Both enabled is how we're doing it for the source builds.